car.py

```diff
@@ -1,129 +1,50 @@
-import dynamics
-import aaron_array
-import electronics
+import battery
+import solar_array
 import motor
-import weather
-import datetime
-import battery_pack
-import battery_module
+
 import time
-import track
-import math
+import datetime
 
-# model of the solar car
-# needs main module to initialize and create race strategy
+import numpy as np
+
+from track import Track
 class Car:
-    def __init__(self, capacity, voltageOfaSingeCell, latitude, longitude, startingTime): #attributes to initialize once then automatically update
-        self.voltageOfaSingeCell = voltageOfaSingeCell
-        self.capacity = capacity
-        self.totalCapacity = capacity
-        self.weather_conditions = weather.Weather(latitude,longitude, startingTime)
-        self.elec = electronics.Electronics()
-        self.mot = motor.Motor()
-        self.prevTime = startingTime
-        self.deltaTime = 0
+    def __init__(self, track: Track, time: int) -> None:
+        max_capacity = 5400 # Wh
```

Comment:
Make this a constant up top. i.e. something like `MAX_BATTERY_CAPACITY_WH = 5400`, and then `max_capacity = MAX_BATTERY_CAPACITY_WH`